package.json

```diff
@@ -3,11 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "firebase": "^7.4.0",
```

Comment:
not sure that it's right to remove  firebase deps ...

Reply:
Rolled back.